File: test/fixtures/extractor-repo/src/elixir/panel_service.ex
defmodule Acme.Panels.PanelService do
  alias Acme.Panels.PanelClient
  require Logger

  @behaviour Acme.Panels.Loader
  @callback load_panel(String.t()) :: String.t()
  Record.defrecord(:panel, id: nil)

  def load_panel(id) do
    PanelClient.fetch(id)
  end

  defp normalize_id(id) do
    id
  end
end

File: test/fixtures/extractor-repo/src/erlang/panel_service.erl
-module(panel_service).
-include("panel.hrl").
-import(panel_client, [fetch/1]).
-export([load_panel/1]).
-behaviour(gen_server).
-record(panel, {id}).
-callback load_panel(binary()) -> binary().

load_panel(Id) ->
  panel_client:fetch(Id).

normalize_id(Id) ->
  Id.
